### PR TITLE
updating tagoio sdk

### PR DESCRIPTION
**Pull request by rafaelTelesS**
## What does PR do?

The TagoIO Sdk has been updated to work with the new access rule system for analysis. This system allows you to control who can access your analysis, what actions they can perform, and under what conditions. To use the new system, you need to update your Sdk version and follow the documentation on how to create and manage access rules. 

## Type of alteration

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [x] Documentation update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,34 @@ or
 
 You can run locally, you should have last node version and install all dependencies running `npm install` on your terminal in this project folder,
 after that, you can run the project just calling analysis, `node analysis.js`;
```
Comment (Freddyminu) [inline README.md]: Run it locally, you should have lastest node version and install all the dependencies by running `npm install` in your terminal in the project's directory,
after that, you can run the project by calling the analysis, `node analysis.js`;

File: README.md [hunk 2]
```diff
@@ -6,3 +6,34 @@ or
...
+
+## What this does
```
Comment (Freddyminu) [inline README.md]: ## What the analysis does

File: README.md [hunk 3]
```diff
@@ -6,3 +6,34 @@ or
...
+
+This analysis must run by Time Interval. It checks if devices with given Tags had communication in the past minutes. If not, it sends an email or sms alert.
```
Comment (Freddyminu) [inline README.md]: This analysis must run by triggered by a [Time Interval](https://help.tago.io/portal/en/kb/articles/244-trigger-by-schedule). It checks whether devices with given Tags checked in, in the past minutes. If not, it sends an email or sms alert.

File: README.md [hunk 4]
```diff
@@ -6,3 +6,34 @@ or
...
+
+## How to use on TagoIO
```
Comment (Freddyminu) [inline README.md]: ## How to use this analysis internally at TagoIO servers

File: README.md [hunk 5]
```diff
@@ -6,3 +6,34 @@ or
...
+
+    1 - checkin_time: Minutes between the last input of the device before sending the notification.
+
+    2 - tag_key: Device tag Key to filter the devices.
+
+    3 - tag_value: Device tag Value to filter the devices.
+
+    4 - email_list: Email list comma separated.
+
+    5 - sms_list: Phone number list comma separated. The phone number must include the country code
```
Comment (Freddyminu) [inline README.md]: 5 - sms_list: Phone number list comma separated. The phone number must include the country code.

File: README.md [hunk 6]
```diff
@@ -6,3 +6,34 @@ or
...
+
+
+Steps to add a new policy:
+
+   1 - Click the button "Add Policy" at this url: https://admin.tago.io/am;
```
Comment (Freddyminu) [inline README.md:33]: 1 - Click the button "[Add Policy](https://admin.tago.io/am)".

File: README.md [hunk 7]
```diff
@@ -6,3 +6,34 @@ or
...
+
+   2 - In the Target selector, select the Analysis with the field set as "ID" and choose your Analysis in the list;
```
Comment (Freddyminu) [inline README.md]: 2 - In the Target selector, with the field set as "ID", choose your Analysis in the list;

File: analysis.js [hunk 8]
```diff
@@ -8,22 +8,21 @@
...
- ** Steps to generate an account_token:
- ** 1 - Enter the following link: https://admin.tago.io/account/
- ** 2 - Select your Profile.
- ** 3 - Enter Tokens tab.
- ** 4 - Generate a new Token with Expires Never.
- ** 5 - Press the Copy Button and place at the Environment Variables tab of this analysis.
+ ** How to use:
+ ** To analysis works, you need to add a new policy in your account. Steps to add a new policy:
+ **  1 - Click the button "Add Policy" at this url: https://admin.tago.io/am;
+ **  2 - In the Target selector, select the Analysis with the field set as "ID" and choose your Analysis in the list;
```
Comment (Freddyminu) [inline analysis.js]: ** 2 - In the Target selector, with the field set as "ID", choose your Analysis in the list;

**Review by Freddyminu — CHANGES_REQUESTED** [review]

**Review by vitorfdl — APPROVED** [review]

**Pull request merged**